ui/app/components/app/permissions-connect-header/index.scss

```diff
@@ -27,13 +28,16 @@
     @extend %header--24;
     text-align: center;
     color: $Black-100;
-    margin-top: 16px;
+    margin-top: 14px;
   }
 
-  &__text {
+  &__text, &__subtitle {
     @extend %content-text;
     text-align: center;
-    margin-top: 4px;
     color: $Grey-500;
   }
+
+  &__subtitle {
```

Comment:
This class seems to be unused. I'm guessing you forgot to add it to the header component?

Reply:
change was made but file not saved :man_facepalming:

Reply:
Correctly saved in 616009b3c